scripts/ci_build_and_test_in_container.sh

```diff
@@ -208,6 +217,7 @@ or_die python3 scripts/config-build.py \
                -DBLT_MPI_COMMAND_APPEND='"--allow-run-as-root;--oversubscribe"' \
                -DGEOSX_INSTALL_SCHEMA=${GEOSX_INSTALL_SCHEMA} \
                -DENABLE_COVERAGE=$([[ "${CODE_COVERAGE}" = true ]] && echo 1 || echo 0) \
+               -DGEOS_PYTHON_PACKAGES_BRANCH:STRING="bugfix/sherman/fixRebaselineLogging" \
```

Comment:
This temporary branch change can be removed.  It may fix an issue on quartz, but only adds some extra error checking/documentation for current problem.